caffe2/CMakeLists.txt

```diff
@@ -1365,6 +1372,12 @@ if(USE_DISTRIBUTED)
       target_compile_definitions(torch_cuda PUBLIC USE_C10D_NCCL)
     endif()
   endif()
+  if(USE_C10D_XCCL)
+    target_compile_definitions(torch_xpu PUBLIC USE_C10D_XCCL)
+    set_source_files_properties(
+      ${TORCH_SRC_DIR}/csrc/distributed/c10d/ProcessGroupXCCL.cpp
+      PROPERTIES COMPILE_DEFINITIONS  "CCL_ENABLE_ZE;CCL_ENABLE_SYCL")
```

Comment:
Those flags are just workaround and will be removed after oneCCL gcc build issue is fixed (target to basekit 2025.0). https://jira.devtools.intel.com/browse/MLSL-3281